base58.c

```diff
@@ -41,26 +43,25 @@ typedef uint32_t b58_almostmaxint_t;
 #define b58_almostmaxint_bits (sizeof(b58_almostmaxint_t) * 8)
 static const b58_almostmaxint_t b58_almostmaxint_mask = ((((b58_maxint_t)1) << b58_almostmaxint_bits) - 1);
 
-//	MSVC 2017 doesn't support the GCC (and probably clang) extension
-//	for dynamic arrays in C
-#ifdef _WIN32
+//	MSVC 2017 C99 doesn't support dynamic arrays in C
+#ifdef _MSC_VER
 #define b58_log_err(msg, ...) printf("[" __FUNCTION__ " - ERROR]: " msg, __VA_ARGS__)
 
-#define b58_alloc_mem(type, name, count)					\
```

Comment:
I'm honestly not sure what's up with the indentation of the escape slashes here. With invisibles in Atom turned on everything looks as if it's tabbed appropriately, and this is exactly how it looked before the push:

![badformat](https://user-images.githubusercontent.com/911971/40892206-7bece1a8-6748-11e8-99b4-ca55f2bb4476.PNG)